DataFormats/OnlineMetaData/interface/OnlineMetaDataRaw.h

```diff
@@ -0,0 +1,67 @@
+#ifndef DATAFORMATS_ONLINEMETADATA_ONLINEMETADATARAW_H
+#define DATAFORMATS_ONLINEMETADATA_ONLINEMETADATARAW_H
+
+//---------------------------------------------------------------------------
+//!  \class OnlineMetaDataRaw
+//!  \brief Structure of raw data from soft FED 1022
+//!
+//!  \author Remi Mommsen - Fermilab
+//---------------------------------------------------------------------------
+
+#include <cstdint>
+#include "EventFilter/FEDInterface/interface/fed_header.h"
+#include "EventFilter/FEDInterface/interface/fed_trailer.h"
+
+namespace onlineMetaData {
```

Comment:
perhaps ```namespace online``` is good enough?

Reply:
I prefer to keep it specific.

Reply:
cms naming rules ask for all lowercase namespace.
"online" is simpler in this context.

Reply:
Okay, will change it.